_posts/2024-10-01-debian-11.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: post
+title: Debian LTS support in our modules
+date: 2024-10-01
+github_username: bastelfreak
+category: changelog
+---
+
+In the past, Vox Pupuli decided against supporting [Debian LTS timeframes](https://wiki.debian.org/LTS).
+This was mostly because our CI resources were quite limited back in the days of Travis CI.
+Over the time we got multiple requests from individuals and companys to also support Debian LTS, some one them even willing to sponsor us.
```

Comment:
```suggestion
Over the time we got multiple requests from individuals and companies to also support Debian LTS, some of them even willing to sponsor us.
```

Reply:
thanks, that happens when I had too much or not enough caffeine :D